style.css

```diff
@@ -177,14 +216,289 @@
   .story-title ul li {
     display: inline-block;
     background: none;
-    border-right: 1px solid #ccc;
+    border-right: 1px solid #646369;
     padding: 0.3rem 1.5rem;
   }
 
-  .story-title ul li #ruby {
+  .btn-1 {
+    display: inline-block;
+    background: #9490ff;
+    color: #fff;
+    cursor: pointer;
+    border: none;
+    padding: 1.1rem;
+    margin: 1.8rem;
+  }
+
+  .btn-1 :enabled {
+    color: #7d73fc;
+  }
+
+  .btn-1 :hover {
+    color: #7d73fc;
+  }
+
+  .btn-1 :disabled {
+    color: #2a2b33;
+  }
+
+  .sub-grid-5 h2 {
+    font-size: 32px;
+    line-height: 44px;
+    color: #9390ff;
+    margin-top: 5rem;
+  }
```

Comment:
- [ ] Please use  flexbox  to layout the elements of the form.
- [ ] Pelase add Button interactions such as hover, active (when it's pressed), disabled and enable (default state) styled as shown in the Figma template.:
<img width="619" alt="Screenshot 2022-03-30 at 12 35 46" src="https://user-images.githubusercontent.com/77038610/160788560-a2f2dec4-8a6a-4027-95ce-ad06cb5bd16d.png">